### PR TITLE
Update addon-manager to use debian-base:0.3.2.

### LINKED ISSUE #68858: Update kube-addon-manager base Alpine image to 3.8.1
**Is this a BUG REPORT or FEATURE REQUEST?**:

/kind feature

Alpine fixed a RCE vulnerability in apk in 3.8.1 (https://alpinelinux.org/posts/Alpine-3.8.1-released.html)
Update the base image used for addon-manager (https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/addon-manager/Makefile#L22) to pick it up.

Note: I'd send a PR if it depended directly on alpine image, not bashell/alpine.

### PR DESCRIPTION
**Pull request by MrHohn**
**What this PR does / why we need it**:
/assign @awly @ixdy

This is similar to https://github.com/kubernetes/kubernetes/pull/67283.

The actual image bump will happen separately.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #68858 

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cluster/addons/addon-manager/Makefile [hunk 1]
```diff
@@ -15,24 +15,10 @@
 IMAGE=staging-k8s.gcr.io/kube-addon-manager
 ARCH?=amd64
 TEMP_DIR:=$(shell mktemp -d)
-VERSION=v8.7
+VERSION=v8.8
```
Comment (ixdy) [inline cluster/addons/addon-manager/Makefile]: is there any significance to the major/minor of this version?

File: cluster/addons/addon-manager/Dockerfile [hunk 2]
```diff
@@ -14,6 +14,8 @@
 
 FROM BASEIMAGE
 
+RUN apt-get update && apt-get install -y bash
```
Comment (ixdy) [inline cluster/addons/addon-manager/Dockerfile]: use `clean-install bash` instead. (it's a helper script we installed in `debian-base` which does this and also cleans up the caches afterwards.)

File: cluster/addons/addon-manager/Makefile (hunk 1 above)
Comment (MrHohn) [reply cluster/addons/addon-manager/Makefile]: Not really, I should probably call it `v8.7-r1`?

File: cluster/addons/addon-manager/Dockerfile (hunk 2 above)
Comment (MrHohn) [reply cluster/addons/addon-manager/Dockerfile]: Thanks, good to know. Updated now.

File: cluster/addons/addon-manager/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
+### Version 8.7-r1  (Mon October 1 2018 Zihong Zheng <zihongz@google.com>)
```
Comment (ixdy) [inline cluster/addons/addon-manager/CHANGELOG.md]: eh, `v8.8` was fine. wasn't sure whether it merited bumping to v9, but I guess not.

Comment (MrHohn) [reply cluster/addons/addon-manager/CHANGELOG.md]: Ah got it. Reverted to use `v8.8`.

**Comment by MrHohn** [comment]
Thanks, adding @mikedanese for approval.
/assign @mikedanese

**Pull request merged**